romanesco/plugins/girder_io/__init__.py

```diff
@@ -44,27 +48,37 @@ def fetch_handler(spec, **kwargs):
     else:
         raise Exception('Invalid resource type: ' + resource_type)
 
-    return dest
+    if target == 'filepath':
+        return dest
+    elif target == 'memory':
+        with open(dest, 'rb') as fd:
+            return fd.read()
+    else:
+        raise Exception('Invalid Girder push target: ' + target)
 
 
 def push_handler(data, spec, **kwargs):
     parent_type = spec.get('parent_type', 'folder')
-    name = spec.get('name', os.path.basename(data))
-    description = spec.get('description')
+    taskOutput = kwargs.get('task_output', {})
+    target = taskOutput.get('target', 'filepath')
```

Comment:
You are setting the default here to filepath - for other modes it seems to make sense to have memory be the default (see my PR at https://github.com/Kitware/romanesco/pull/184/files#diff-792252c08973496d9a93bda90b861811R14). Are we ok with explaining this difference in defaults in the docs?